notebooks/notebook2image/ODSC19-Demo-OpenShift-Machine-Learning-Setup.ipynb

```diff
@@ -107,6 +107,17 @@
     "Once the user has made the appropriate changes to their notebook they can push it back to github for the s2i builder. "
```

Comment:
<p>nit: Do you want to add "Set env variables" to the list of items here? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/data-science-workflow-examples/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added :+1: